src/main/java/jmgomez/apipolicy/model/Accident.java

```diff
@@ -9,8 +9,6 @@
 @NoArgsConstructor
 @AllArgsConstructor
 public class Accident {
-    @JsonProperty("siniestroId")
-    private String sinisterId;
-    @JsonProperty("estado")
-    private String status;
+    private String siniestroId;
```

Comment:
Why have we reverted this? It was better before. Let jackson do that naming mapping and keep your classes in English if it's possible